src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1570,6 +1575,11 @@ public function testGetEncodings()
         $request = new Request();
         $request->headers->set('Accept-Encoding', 'gzip;q=0.4,deflate;q=0.9,compress;q=0.7');
         $this->assertEquals(['deflate', 'compress', 'gzip'], $request->getEncodings());
+
+        $request = new Request();
+        $request->headers->set('Accept-Encoding', '123');
+        $this->assertIsString($request->getEncodings()[0]);
+        $this->assertEquals(['123'], $request->getEncodings());
```

Comment:
assertSame everywhere?

Reply:
Tried to keep the same code style as the existing tests.

Should I just change the two lines (`assertIsString` and `assertEquals`) to `assertSame`?

Reply:
yes please

Reply:
I think so

Reply:
Also, I suggest splitting that into a new test case instead of adding one more nested test. This makes it a lot more clear what the test is covering (as it will have a name) and allows it to run and fail separately (as it is a separate test). This would follow the PHPUnit best practices (and yes, existing tests don't follow that, but HttpFoundation is one of the first Symfony components so its tests were written at a time when using PHPUnit was still new in the Symfony community, so not following best practices yet).

Reply:
Done in 6ba174c39188971fcebdaf0f7bc020c670b789ca